src/functionalTest/resources/scenarios/RIA-5727-end-appeal-confirmation.json

```diff
@@ -0,0 +1,20 @@
+{
+  "description": "RIA-5727 end appeal automatically confirmation",
+  "request": {
+    "uri": "/asylum/ccdSubmitted",
+    "credentials": "CaseOfficer",
+    "input": {
+      "eventId": "endAppealAutomatically",
+      "state": "submitAppeal",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json"
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "confirmation": {
+      "header": "# Appeal has been ended"
+    }
+  }
+}
```

Comment:
Not sure if it's needed, but it's flagged up without a new line at end of the test?

Reply:
Added new line at end of file.